plugins/SmfImport/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Importer for SMF-like file format, such as .mid, .ove, .wrk
+
+INCLUDE(BuildPlugin)
+
+# check for drumstick-file
+PKG_CHECK_MODULES(DRUMSTICK_FILE REQUIRED drumstick-file>=1.1.0)
+IF(NOT DRUMSTICK_FILE_FOUND)
+        MESSAGE(FATAL_ERROR "LMMS requires libdrumstick-file and drumstick-dev >= 1.0.11 - please install, remove CMakeCache.txt and try again!")
```

Comment:
Looks like the as of now unreleased Ubuntu 17.04 will have libdrumstick 1.0.2. Is there something in 1.0.11 that is absolutely required? (https://launchpad.net/ubuntu/+source/libdrumstick)